govc/test/vm.bats

```diff
@@ -464,6 +464,19 @@ load test_helper
   assert_failure
 }
 
+@test "vm.migrate" {
+  vcsim_env
+  vm=$(new_empty_vm)
+
+  # migrate from H0 to H1
+  run govc vm.migrate -host DC0_C0/DC0_C0_H1 "$vm"
```

Comment:
this might be a bit unclear, should we change this parameter to -targetHost or something as all the other commands reference the source host when using that parameter?